ports/libfido2/vcpkg.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "libfido2",
+  "version-string": "1.6.0",
```

```suggestion
  "version": "1.6.0",
```
We prefer to use a proper package version to describe it. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions